orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -41,15 +42,23 @@ def __init__(self, module):
         the onnx graph, and ExecutionAgent to run the onnx graph.
         """
 
-        # Original and flattened (tranformed) output module
-        self._original_module = module._original_module
-        self._flattened_module = module
-
-        # Exported model
+        self._original_module = None
         self._onnx_model = None
+        self._flattened_module = None
+
+        if isinstance(module, torch.nn.Module): 
+            # Original and flattened (tranformed) output module
+            self._original_module = module._original_module
+            self._flattened_module = module
+            # TODO: Single device support for now
+            self._device = _utils.get_device_from_module(module)
+        elif isinstance(module, onnx.ModelProto):
+            # Exported model
+            self._onnx_model = module
+            # Model after inference optimization or gradient building.
+            self._optimized_onnx_model = module
+            self._onnx_model_parameters = onnx_model_parameters
```

Comment:
maybe you could iterate over `module.graph.initializer` and fill in this without user interaction?

Reply:
yep, thanks. No reason for execution manager to know about model parameters